hashing/te-tag-query-python/all_in_one_lib/content_type/meta.py

```diff
@@ -0,0 +1,21 @@
+##!/usr/bin/env python
+
+"""
+Stores all the content type implementations for lookup
+"""
+
+import typing as t
+
+from . import base, text, video, photo
+from ..signal_type import base as signal_base
```

Comment:
TODO: Rename all these base.py 's to a more meanginful name, signal_base, content_base, etc.

Reply:
Decided to punt this to a followup PR